pkg/jobrunaggregator/jobrunaggregatorapi/fs_jobrun.go

```diff
@@ -0,0 +1,213 @@
+package jobrunaggregatorapi
+
+import (
+	"context"
+	"encoding/xml"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/openshift/ci-tools/pkg/junit"
+
+	utilerrors "k8s.io/apimachinery/pkg/util/errors"
+	prowjobv1 "k8s.io/test-infra/prow/apis/prowjobs/v1"
+)
+
+type fsJobRun struct {
+	workingDirectory string
+
+	jobName        string
+	jobRunID       string
+	gcsProwJobPath string
+	gcsJunitPaths  []string
+}
+
+func NewFilesystemJobRun(workingDirectory, jobName, jobRunID string) (JobRunInfo, error) {
```

Comment:
If all this code is unused, I'd propose we play the YAGNI card and ditch it. This is a decent chunk of code for someone to come up to speed on and the less there is of it, the beter. 

A separate commit removing it would be fine if we'd like to keep the historical record in case we want it later.